spec/commands/remove_spec.rb

```diff
@@ -44,6 +44,25 @@
           source "#{file_uri_for(gem_repo1)}"
         G
       end
+
+      context "when gem is specified in multiple lines" do
+        it "shows success for removed gem" do
+          gemfile <<-G
+            source '#{file_uri_for(gem_repo1)}'
+
+            gem 'rack',
```

Comment:
Can you add the extra gems that are the original report?

Like this:

```
gem 'git'
gem 'some-gem',
    git: 'ssh://gitlab-server.com/project/repo.git',
    branch: 'master'
gem 'nokogiri'
```

And add expectation for git and nokogiri.

Reply:
thanks for the feedback, sure would give more test context, will do soonish ;)